examples/boussinesq_2d_imex/playground.py

```diff
@@ -36,12 +36,12 @@
     swparams['do_LU'] = True
 
     sparams = {}
-    sparams['maxiter'] = 12
+    sparams['maxiter'] = 16
```

Comment:
What would be a reasonable tolerance to get approx. 10-20 iterations per block, then? Does this change during the simulation or is it more or less constant?